models/yang/sonic/sonic-bgp-common.yang

```diff
@@ -68,16 +68,6 @@ module sonic-bgp-common {
 
   grouping sonic-bgp-cmn {
 
-    leaf vrf_name {
-      type string;
-        description "Network-instance/VRF name";
-    }
-
-    leaf peer_group_name {
```

Comment:
As peer_group_name is common to both NBR and peer group, looks like its missed to add it in nbr table

Reply:
Added this attribute to SONIC_BGP_NEIGHBOR table